app/lib/irs1040_pdf.rb

```diff
@@ -15,15 +15,12 @@ def initialize(submission)
   end
 
   def hash_for_pdf
-    urbanization = @address.urbanization
-    urbanization += "," if urbanization.present?
-
     answers = {
       FilingStatus: @xml_document.at("IndividualReturnFilingStatusCd")&.text,
       PrimaryFirstNm: @intake.primary_middle_initial.present? ? "#{@intake.primary_first_name} #{@intake.primary_middle_initial}" : @intake.primary_first_name,
       PrimaryLastNm: @intake.primary_last_name,
       PrimarySSN: @xml_document.at("PrimarySSN")&.text,
-      AddressLine1Txt: [urbanization, @address.street_address, @address.street_address2].compact.join(" "),
+      AddressLine1Txt: [@address.urbanization, @address.street_address, @address.street_address2].compact.join(" "),
```

Comment:
nice. im glad this pattern is mostly working out without getting too gnarly working out.

Reply:
I do wonder if at some point soon we should change it so the `Address` class computes this. I know the XML truncates at 35, and the PDF doesn't; it would be good to make those behave the same anyway.

Reply:
I don't really want to do that in this pull request, though!